ddtrace/contrib/internal/ray/patch.py

```diff
@@ -188,23 +189,29 @@ def traced_submit_job(wrapped, instance, args, kwargs):
 
     submission_id = kwargs.get("submission_id") or generate_job_id()
     kwargs["submission_id"] = submission_id
+    job_name = kwargs.get("metadata", {}).get("job_name", "")
 
     # Root span creation
-    job_span = tracer.start_span("ray.job", service=submission_id, span_type=SpanTypes.RAY)
+    job_span = tracer.start_span("ray.job", service=job_name or get_dd_job_name(submission_id), span_type=SpanTypes.RAY)
```

Comment:
It is a bit of a nit but we are doing twice job_name or get_dd_job_name. It might be worth to check if job_name is "" and if it is the case, replace it with get_dd_job_name(submission_id)